.circleci/config.yml

```diff
@@ -122,7 +123,6 @@ workflows:
           chart: policies-shared
           context: "architect"
           explicit_allow_chart_name_mismatch: true
-          on_tag: false
```

Comment:
Wait - shouldn't it still work because the `collection` requires this?
```
requires:
  - push-policies-shared-to-catalog
  ```

Reply:
Well, the collection will not run until this runs but this does not run on tags and collections only runs on tags

Reply:
https://app.circleci.com/pipelines/github/giantswarm/kyverno-policies There is no workflow for the tag here :(

Reply:
Okay I follow now. Should we remove it everywhere when we push to catalog then?

Reply:
Yes maybe :D I do not see why we do not to push on tag anyway :D